spec/models/auth_token_spec.rb

```diff
@@ -2,14 +2,18 @@
   include TokenHelper
 
   describe "#data" do
-    it "returns the data hash when valid" do
-      token = AuthToken.new(encrypt_and_sign_token(data: { a: "b" }))
-      expect(token.data).to eq(a: "b")
+    context "with a SHA1 token" do
+      it "returns the data hash when valid" do
```

Comment:
Nitpick: we tend to avoid using `context` blocks unless there's more than one `it` block inside it. In this case, we'd usually just do:

```suggestion
    it "returns the data hash when given a valid SHA1 token" do
```